pulseguardian/web.py

```diff
@@ -277,40 +279,65 @@ def bindings_listing(queue_name):
 
 # Authentication related
 
-@app.route('/auth/login', methods=['POST'])
-def auth_handler():
-    # The request has to have an assertion for us to verify
-    if 'assertion' not in request.form:
-        return jsonify(ok=False, message="Assertion parameter missing")
+@app.route('/auth/callback')
+def callback_handling():
+    """
+    Callback from Auth0
 
-    # Send the assertion to Mozilla's verifier service.
-    data = dict(assertion=request.form['assertion'],
-                audience=config.persona_audience)
-    resp = requests.post(config.persona_verifier, data=data, verify=True)
+    Auth0 will call back to this endpoint once it has acquired a user in some
+    capacity (from Google, or their email).  Here we verify what we got back
+    it consistent with Auth0, then match what we got back
```

Comment:
This comment could be cleaned up a bit: it's missing an "if", using "it" instead of "is", and using "what we got back" twice in one sentence. :)